tests/test_client.py

```diff
@@ -1,6 +1,6 @@
 import re
 import json
-from six.moves.urllib.parse import parse_qs
+from six.moves.urllib.parse import parse_qs  # noqa
```

Comment:
This was breaking the Travis CI build. Below is the Travis CI error:

________________________ [pylint] tests/test_client.py _________________________
W:  3, 0: Relative import 'six.moves.urllib.parse', should be 'six.moves' (relative-import)

The error seems to be an open pylint issue. See https://github.com/PyCQA/pylint/issues/2180

Reply:
I suggested switching from Pylint to Black in #108. That would resolve the problem, as well 😄

Reply:
haha good to know! Maybe one day lol